system/config/templates/features/aca_individual_market/aca_individual_market.yml

```diff
@@ -4,10 +4,10 @@ registry:
       - :fdsh_gateway
       - :aca_individual_market
     features:
-      - key: :h41_renewal_transmissions
-        is_enabled: <%= ENV['H41_RENEWAL_TRANSMISSIONS_IS_ENABLED'] || false %>
+      - key: :h41_renewal_transmissions_creation
+        is_enabled: <%= ENV['H41_RENEWAL_TRANSMISSIONS_CREATION_ENABLED'] || false %>
```

Comment:
This env var is missing the conventional `IS_ENABLED` suffix.  Right now it just ends with `ENABLED`.